app/templates/releases/canary.hbs

```diff
@@ -19,3 +19,21 @@ npm install --save-dev https://s3.amazonaws.com/builds.emberjs.com{{model.canary
   project=model.emberData
   channel="canary"}}
 </div>
+
+<h2>What is canary?</h2>
+
+<p>
+  "Canary" is whatever code is on the main branch of an Ember project.
+  It is unreleased, and is not recommended for production or new learners.
+  Often, features in canary are a work-in-progress.
+  They may change substantially or be removed before being released,
+  and in rare cases, they may never be released.
+</p>
+
+<p>
+  Canary is used in many test suites, so that any bugs in the framework code
+  are detected quickly and fixed before a beta is released.
+  Some of Ember's key contributors make a habit of working off beta and canary,
+  and even use it in production,
```

Comment:
I understand the mention, but saying it is used in production undermines the previous paragraph a little bit.

Reply:
removed this sub-clause